checkov/terraform/checks/resource/aws/RDSCACertIsRecent.py

```diff
@@ -21,7 +21,7 @@ def get_inspected_key(self) -> str:
         return "ca_cert_identifier"
 
     def get_expected_values(self) -> List[Any]:
-        return ["rds-ca-2019"]
+        return ["rds-ca-2019", "rds-ca-rsa2048-g1", "rds-ca-rsa4096-g1", "rds-ca-ecc384-g1"]
```

Comment:
```suggestion
        return ["rds-ca-rsa2048-g1", "rds-ca-rsa4096-g1", "rds-ca-ecc384-g1"]
```
let's remove the 2019 version, because it is anyway recommended to use `rds-ca-rsa2048-g1` instead

Reply:
That's actually a good idea. On top of that, `rds-ca-2019` does not support automatic rotation contrary to the others 🙂

Reply:
Fixed in 68787e3275d972a5293b0fd9c85eaac36d6f870c